src/ceph-volume/plugin/zfs/ceph_volume_zfs/util/__init__.py

```diff
@@ -1 +1,84 @@
-# -*- coding: utf-8 -*-
+import logging
```

Comment:
all of these utils can be reused from ceph-volume

Reply:
@alfredodeza 
Can I include an __init__.py file from a competely different module??